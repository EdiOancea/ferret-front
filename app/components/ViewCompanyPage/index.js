import React from 'react';
import PropTypes from 'prop-types';
import ViewHeadlineOutlinedIcon from '@material-ui/icons/ViewHeadlineOutlined';

import PaperFrame from 'components/PaperFrame';
import ViewCompanyContainer from 'containers/ViewCompanyContainer';

const ViewCompanyPage = props => (
  <PaperFrame
    headerText="Company Information"
    icon={<ViewHeadlineOutlinedIcon />}
  >
    <ViewCompanyContainer slug={props.match.params.slug} />
  </PaperFrame>
);

ViewCompanyPage.propTypes = {
  match: PropTypes.shape({
    params: PropTypes.shape({
      slug: PropTypes.number.isRequired,
    }),
  }),
};

export default ViewCompanyPage;
