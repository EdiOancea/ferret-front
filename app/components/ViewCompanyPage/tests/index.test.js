import React from 'react';
import { shallow } from 'enzyme';

import ViewCompanyPage from 'components/ViewCompanyPage';

describe('<ViewCompanyPage />', () => {
  it('should not regress', () => {
    const config = {
      params: {
        slug: '1',
      },
    };
    const tree = shallow(
      <ViewCompanyPage match={config}>
        <div> a </div>
      </ViewCompanyPage>,
    );
    expect(tree.children.length).toBe(1);
    expect(tree).toMatchSnapshot();
  });
});
