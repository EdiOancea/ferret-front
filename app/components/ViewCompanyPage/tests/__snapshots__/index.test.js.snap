// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ViewCompanyPage /> should not regress 1`] = `
<WithStyles(PaperFrame)
  headerText="Company Information"
  icon={<pure(ViewHeadlineOutlinedIcon) />}
>
  <withSaga(withReducer(Connect(ViewCompanyContainer)))
    slug="1"
  />
</WithStyles(PaperFrame)>
`;
